apollo-client/src/main/java/com/ctrip/framework/apollo/internals/SimpleConfig.java

```diff
@@ -37,18 +39,23 @@
  */
 public class SimpleConfig extends AbstractConfig implements RepositoryChangeListener {
   private static final Logger logger = LoggerFactory.getLogger(SimpleConfig.class);
+  private final String m_appId;
   private final String m_namespace;
   private final ConfigRepository m_configRepository;
   private volatile Properties m_configProperties;
   private volatile ConfigSourceType m_sourceType = ConfigSourceType.NONE;
 
+  private ConfigUtil m_configUtil = ApolloInjector.getInstance(ConfigUtil.class);
```

Comment:
As there is a new `m_appId` field, there is no need to declare the `m_configUtil` as a global variable.